.drone.yml

```diff
@@ -28,7 +28,7 @@ build:
       - tests/wait_until_previous_builds_complete.sh
 
   vic:
-    image: golang:1.8
+    image: $${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.22}
```

Comment:
just curios, what is eminent-nation-87317? If that's the name of the repo, can't we rename it to something else?